### PR TITLE
Tested Seeding Input Type Defaults

### LINKED ISSUE #156: Test Seeding Input Type Defaults
The tests created for this issue will test the default contents of the "Seeding Type" (Tray/Direct) section of the Seeding Input Form in the FieldKit.

These tests must check that the Seeding Type (Tray/Direct) section of the Seeding Input Form:
- [ ] - has an element for "Tray" that is enabled.
- [ ] - has an element for "Direct" that is enabled.
- [ ] - neither the "Tray" nor the "Direct" element is selected.
- [ ] - A message is visible indicating that "Tray" or "Direct" must be selected.
- [ ] - The form elements for either the "Tray" and "Direct" seedings are not visible or do not exist.

Other information:
 - The source code and tests for the Seeding Input Form will be in the `farmdata2/farmdata2_modules/fd2_field_kit/seedingInput` directory.
 - The `seedingInput.html` file contains the source code for the page.
 - The tests for this issue should be created in a new file named `seedingInput.type.defaults.spec.js`
 - The above tests should be broken up into logically organized `it` blocks within the test file.

### PR DESCRIPTION
**Pull request by pranavm2109**
__Pull Request Description__

This pull request checks the type defaults of the "Tray" and "Direct" elements in the Seeding Type section of the Seeding Input Form. It starts by checking whether the "Tray" and "Direct" elements are enabled, and proceeds to confirm that neither of them are selected. The pull request culminates by confirming the presence of a message stating that either the "Tray" or the "Direct" elements must be selected, and that the form elements for either of the two seeding types are not visible or do not exist.
Closes #156 

---
__Licensing Certification__

FarmData2 is a [Free Cultural Work](https://freedomdefined.org/Definition) and all accepted contributions are licensed as described in the LICENSE.md file. This requires that the contributor holds the rights to do so. By submitting this pull request __I certify that I satisfy the terms of the [Developer Certificate of Origin](https://developercertificate.org/)__ for its contents.

### CODE REVIEW

File: farmdata2/farmdata2_modules/fd2_field_kit/seedingInput/seedingInput.html [hunk 1]
```diff
@@ -25,7 +25,7 @@ <h1 style='text-align:center;'>Seeding Input Log</h1>
                 <input data-cy='direct-seedings' type='radio' name='typeOfSeeding' value='Direct Seedings'v-model='selectedSeedingType' :checked='selectedSeedingType == "Direct Seedings"' v-bind:disabled='submitInProgress'><label>Direct</label> 
             </legend>
             <div v-show='(selectedSeedingType=="")' style='text-align:center;font-size: medium;'>
-                <p>Please Select Tray Seeding or Direct Seeding<label style='color: #da4f3f'>*</label></p>
+                <p data-cy="message-to-prompt-selection">Please Select Tray Seeding or Direct Seeding<label style='color: #da4f3f'>*</label></p>
```
Comment (braughtg) [inline farmdata2/farmdata2_modules/fd2_field_kit/seedingInput/seedingInput.html:28]: The value of this `data-cy` may be a little long without adding much meaning.  For example, what is it prompting a selection of?  Maybe there is a shorter, but more descriptive value that could be used?

File: farmdata2/farmdata2_modules/fd2_field_kit/seedingInput/seedingInput.type.defaults.spec.js [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+        cy.login("manager1", "farmdata2")
+        cy.visit("/farm/fd2-field-kit/seedingInput")
+    })
+      
+    it("Tests whether Tray and Direct elements are enabled", () => {
+        cy.get("[data-cy='tray-seedings']").should('be.enabled')
+        cy.get("[data-cy='direct-seedings']").should('be.enabled')
+    })
+
+    it("Tests that neither the Tray nor the Direct element is selected", () => {
```
Comment (braughtg) [inline farmdata2/farmdata2_modules/fd2_field_kit/seedingInput/seedingInput.type.defaults.spec.js:22]: Seems like this test and the next could reasonably be combined into one test.

File: farmdata2/farmdata2_modules/fd2_field_kit/seedingInput/seedingInput.type.defaults.spec.js [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+        cy.get("[data-cy='tray-seedings']").should('not.be.selected')
+        cy.get("[data-cy='direct-seedings']").should('not.be.selected')
+    })
+
+    it("Tests that the message indicating that the Tray or Direct element must be selected is visible", () => {
```
Comment (braughtg) [inline farmdata2/farmdata2_modules/fd2_field_kit/seedingInput/seedingInput.type.defaults.spec.js:27]: Combine with the previous test.

**Review by braughtg — CHANGES_REQUESTED** [review]
Please see comments below regarding requested changes.

**Comment by andrewscheiner53** [comment]
@braughtg We believe we have addressed all of your suggestions and are ready for review. Thanks!

**Comment by braughtg** [comment]
Nicely done!

**Pull request merged**